### ISSUE TITLE
[InvalidArgumentException] "\getPageSizeFromFormat()," is not a valid Fqsen.

### ISSUE THREAD

**Issue by alphadevx** [ApiGen/ApiGen#1053]
My environment:

```
$ php -v
PHP 7.2.8 (cli) (built: Jul 17 2018 05:35:43) ( NTS )
```

```
$ cat composer.json 
{
    "require": {
        "apigen/apigen": "dev-master",
        "roave/better-reflection": "dev-master#ce9d784"
    }
}
```

After increasing my memory limit to 2GB I finally thought I could generate the docs for my framework until this happened, now I am not sure where to go next debugging this:

```
$ vendor/bin/apigen generate --destination ~/Documents/alpha/api/3.0.0 ~/Git/alpha/Alpha/
Parsing reflections (this may take a while)...
In Fqsen.php line 48:
                                                     
  [InvalidArgumentException]                         
  "\getPageSizeFromFormat()," is not a valid Fqsen.  
                                                     

Exception trace:
 phpDocumentor\Reflection\Fqsen->__construct() at /home/john/Git/apigen/vendor/phpdocumentor/type-resolver/src/FqsenResolver.php:70
 phpDocumentor\Reflection\FqsenResolver->resolvePartialStructuralElementName() at /home/john/Git/apigen/vendor/phpdocumentor/type-resolver/src/FqsenResolver.php:32
 phpDocumentor\Reflection\FqsenResolver->resolve() at /home/john/Git/apigen/vendor/apigen/apigen/packages/Reflection/src/DocBlock/Tags/See.php:59
 ApiGen\Reflection\DocBlock\Tags\See::create() at n/a:n/a
 call_user_func_array() at /home/john/Git/apigen/vendor/phpdocumentor/reflection-docblock/src/DocBlock/StandardTagFactory.php:201
 phpDocumentor\Reflection\DocBlock\StandardTagFactory->createTag() at /home/john/Git/apigen/vendor/phpdocumentor/reflection-docblock/src/DocBlock/StandardTagFactory.php:122
 phpDocumentor\Reflection\DocBlock\StandardTagFactory->create() at /home/john/Git/apigen/vendor/phpdocumentor/reflection-docblock/src/DocBlockFactory.php:231
 phpDocumentor\Reflection\DocBlockFactory->parseTagBlock() at /home/john/Git/apigen/vendor/phpdocumentor/reflection-docblock/src/DocBlockFactory.php:96
 phpDocumentor\Reflection\DocBlockFactory->create() at /home/john/Git/apigen/vendor/apigen/apigen/packages/Reflection/src/DocBlock/DocBlockFactory.php:41
 ApiGen\Reflection\DocBlock\DocBlockFactory->createFromBetterReflection() at /home/john/Git/apigen/vendor/apigen/apigen/packages/Reflection/src/Transformer/BetterReflection/Class_/ClassMethodReflectionTransformer.php:44
 ApiGen\Reflection\Transformer\BetterReflection\Class_\ClassMethodReflectionTransformer->transform() at /home/john/Git/apigen/vendor/apigen/apigen/packages/Reflection/src/TransformerCollector.php:73
 ApiGen\Reflection\TransformerCollector->transformSingle() at /home/john/Git/apigen/vendor/apigen/apigen/packages/Reflection/src/TransformerCollector.php:43
 ApiGen\Reflection\TransformerCollector->transformGroup() at /home/john/Git/apigen/vendor/apigen/apigen/packages/Reflection/src/Reflection/Class_/ClassReflection.php:404
 ApiGen\Reflection\Reflection\Class_\ClassReflection->getOwnMethods() at /home/john/Git/apigen/vendor/apigen/apigen/packages/Element/src/Cache/ReflectionWarmUpper.php:35
 ApiGen\Element\Cache\ReflectionWarmUpper->warmUpClassReflections() at /home/john/Git/apigen/vendor/apigen/apigen/packages/Element/src/Cache/ReflectionWarmUpper.php:24
 ApiGen\Element\Cache\ReflectionWarmUpper->warmUp() at /home/john/Git/apigen/vendor/apigen/apigen/packages/Reflection/src/Parser/Parser.php:62
 ApiGen\Reflection\Parser\Parser->parseFilesAndDirectories() at /home/john/Git/apigen/vendor/apigen/apigen/src/Application/ApiGenApplication.php:64
 ApiGen\Application\ApiGenApplication->runCommand() at /home/john/Git/apigen/vendor/apigen/apigen/src/Console/Command/GenerateCommand.php:61
 ApiGen\Console\Command\GenerateCommand->execute() at /home/john/Git/apigen/vendor/symfony/console/Command/Command.php:251
 Symfony\Component\Console\Command\Command->run() at /home/john/Git/apigen/vendor/symfony/console/Application.php:946
 Symfony\Component\Console\Application->doRunCommand() at /home/john/Git/apigen/vendor/symfony/console/Application.php:248
 Symfony\Component\Console\Application->doRun() at /home/john/Git/apigen/vendor/symfony/console/Application.php:148
 Symfony\Component\Console\Application->run() at /home/john/Git/apigen/vendor/apigen/apigen/bin/apigen:16

generate [--destination DESTINATION] [--] <source> (<source>)...
```

**Comment by alphadevx** [comment]
I can't get ApiGen 5.x working on PHP 7.2, but did manage to get version 4.x working, there is a temp fork here if it helps unblock anyone:

https://github.com/alphadevx/ApiGen

If I have time I will look at 5.x next.

**Comment by JanTvrdik** [comment]
No longer relevant since https://github.com/ApiGen/ApiGen/releases/tag/v7.0.0-alpha
